mlflow/gateway/config.py

```diff
@@ -44,18 +44,65 @@ def validate_api_key(cls, value):
         return _resolve_api_key_from_input(value)
 
 
+class OpenAIAPIType(str, Enum):
+    OPENAI = "openai"
+    AZURE = "azure"
+    AZUREAD = "azuread"
+
+
 class OpenAIConfig(BaseModel, extra=Extra.allow):
     openai_api_key: str
-    openai_api_type: Optional[str] = None
-    openai_api_base: str = "https://api.openai.com/v1"
+    openai_api_type: str = OpenAIAPIType.OPENAI
```

Comment:
```suggestion
    openai_api_type: OpenAIAPIType = OpenAIAPIType.OPENAI
```

Reply:
Changed the type hint as you suggested :)